pennylane/plugins/expt_tensornet.py

```diff
@@ -51,28 +51,29 @@ class TensorNetwork(Device):
     _capabilities = {"model": "qubit", "tensor_observables": True}
 
     _operation_map = {
-        "QubitStateVector": None,
-        "BasisState": None,
-        "QubitUnitary": unitary,
-        "PauliX": X,
-        "PauliY": Y,
-        "PauliZ": Z,
-        "Hadamard": H,
-        "S": S,
-        "T": T,
-        "CNOT": CNOT,
-        "SWAP": SWAP,
-        "CSWAP": CSWAP,
-        "CZ": CZ,
-        "PhaseShift": Rphi,
-        "RX": Rotx,
-        "RY": Roty,
-        "RZ": Rotz,
-        "Rot": Rot3,
-        "CRX": CRotx,
-        "CRY": CRoty,
-        "CRZ": CRotz,
-        "CRot": CRot3,
+        'BasisState': None,
+        'QubitStateVector': None,
+        'QubitUnitary': unitary,
+        'PauliX': X,
+        'PauliY': Y,
+        'PauliZ': Z,
+        'Hadamard': H,
+        'S': S,
+        'T': T,
+        'CNOT': CNOT,
+        'SWAP': SWAP,
+        'CSWAP': CSWAP,
+        'Toffoli': Toffoli,
+        'CZ': CZ,
+        'PhaseShift': Rphi,
+        'RX': Rotx,
+        'RY': Roty,
+        'RZ': Rotz,
+        'Rot': Rot3,
+        'CRX': CRotx,
+        'CRY': CRoty,
+        'CRZ': CRotz,
+        'CRot': CRot3
```

Comment:
Not sure why this is showing as fully changed, all I did was add the Toffoli gate.